selfdrive/car/hyundai/carstate.py

```diff
@@ -166,7 +172,7 @@ def update(self, cp, cp_cam):
     self.v_wheel_fr = cp.vl["WHL_SPD11"]['WHL_SPD_FR'] * CV.KPH_TO_MS
     self.v_wheel_rl = cp.vl["WHL_SPD11"]['WHL_SPD_RL'] * CV.KPH_TO_MS
     self.v_wheel_rr = cp.vl["WHL_SPD11"]['WHL_SPD_RR'] * CV.KPH_TO_MS
-    self.v_wheel = (self.v_wheel_fl + self.v_wheel_fr + self.v_wheel_rl + self.v_wheel_rr) / 4.
+    self.v_wheel = 1.035 * (self.v_wheel_fl + self.v_wheel_fr + self.v_wheel_rl + self.v_wheel_rr) / 4.
```

Comment:
The stock speed is about 3.8% out, as such, this 3.5% increase makes speedo, speed limit, and cruise speed match.

Reply:
ok for now, would be better to put this in `values.py` (see `honda/values.py`)